api_docs/changelog.md

```diff
@@ -20,6 +20,21 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 242**
+
+* [`GET /streams/{stream_id}`](/api/get-stream-by-id), [`GET /events`]
+  (/api/get-events): The streams and subscriptions now include a `archived`
+  field. This change ensures that deactivated streams are sent back to the
+  client, enabling archived stream and topic links to function correctly.
+
+* [`GET /streams`](/api/get-streams): The parameter `exclude_archived`
+  controls whether archived streams should be returned.
+
+* [`POST /register`](/api/register-queue): The new [client capability]
+  (/api/register-queue#parameter-client_capabilities) `archived_streams`
+  allows the client to decide whether to include archived streams in the
+  result.
```

Comment:
The commit message for this commit still incorrectly talks about a `deactivated` field, when it's called `archived`.

Reply:
Done.